pkg/olm/clusterserviceversionlist.go

```diff
@@ -45,3 +46,36 @@ func ListClusterServiceVersion(
 
 	return csvObjects, nil
 }
+
+// ListClusterServiceVersionWithNamePattern returns a cluster-wide csv inventory filtered by the name pattern.
+func ListClusterServiceVersionWithNamePattern(
+	apiClient *clients.Settings,
```

Comment:
Please format augments according convention> https://github.com/openshift-kni/eco-goinfra#functions-format

func Function(
    argInt1 int,
    argInt2 int,
    argInt3 int,
    argInt4 int,
    argString1 string,
    argString2 string,
    argString3 string,
    argString4 string) {
    ...
}

Reply:
I'm sorry, but I don't understant what is wrong with the original version. And, btw, it is identical to the ListClusterServiceVersion

Reply:
seems like namePattern, nsname string need to be on separate lines, i.e.

```
namePattern string, 
nsname string
```

Reply:
thank you Sasha!